app/assets/javascripts/front/music-player.js

```diff
@@ -126,4 +131,48 @@ class MusicPlayer {
     Util.sendAnalyticsEvent(`Music ${type}`, { globalTitle: `Song ${data.id}`, localTitle: data.name })
   }
 
+  seek(seconds, direction) {
+    const duration = Amplitude.getSongDuration()
+    const currentTime = parseFloat(Amplitude.getSongPlayedSeconds())
+    let targetTime = direction == 'backward' ? currentTime - seconds : currentTime + seconds
+    Amplitude.setSongPlayedPercentage((targetTime / duration) * 100)
+  }
+
+  setupMediaSession() {
+    const data = Amplitude.getActiveSongMetadata()
+    let skipSeconds = 10
+    if ('mediaSession' in navigator) {
+      navigator.mediaSession.metadata = new MediaMetadata({
+        title: data.name,
+        artist: this.artistsList.innerText,
+        artwork: [
+          { src: 'https://dummyimage.com/96x96', sizes: '96x96', type: 'image/png' },
+          { src: 'https://dummyimage.com/128x128', sizes: '128x128', type: 'image/png' },
+          { src: 'https://dummyimage.com/192x192', sizes: '192x192', type: 'image/png' },
+          { src: 'https://dummyimage.com/256x256', sizes: '256x256', type: 'image/png' },
+          { src: 'https://dummyimage.com/384x384', sizes: '384x384', type: 'image/png' },
+          { src: 'https://dummyimage.com/512x512', sizes: '512x512', type: 'image/png' },
```

Comment:
I see that you are using placeholder images right now. What did you implementation using `data.cover_image_url` look like?

Reply:
Simply like `src: data.cover_art_url`.
The `data` is like this:
```
artists: Array(2)
    0: {name: "Leo Vertunni", url: "http://indialucia.com"}
    1: {name: "Shivakumar Sharma", url: "http://www.santoor.com/"}
cover_art_url: "/uploads/artist/image/5/39e74f12ec.jpg"
duration: null
index: 0
live: false
name: "Raag Durga"
url: "/uploads/track/1/music.mp3"
```